### PR TITLE
add option to propagate binaries without access to internet

### PR DESCRIPTION
**Pull request by koroglusaban**
its allow to download binary from own store

### CODE REVIEW

**Comment by paulfantom** [comment]
Please fix issues reported by Travis CI: https://travis-ci.com/cloudalchemy/ansible-node-exporter/jobs/258266272#L280-L288 and add molecule tests. Everything can be found in https://github.com/cloudalchemy/ansible-prometheus/pull/239

File: defaults/main.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+node_exporter_binary_local_dir: ''
```
Comment (paulfantom) [inline defaults/main.yml]: This should be below `node_exporter_version` to make it similar to other repositories in cloudalchemy.

Comment (koroglusaban) [reply defaults/main.yml]: Fixed

**Comment by koroglusaban** [comment]
Styling errors are fixed

**Comment by paulfantom** [comment]
Great job on fixing errors. 

This still needs molecule tests before it is merged.

**Comment by koroglusaban** [comment]
is this molecule check
https://travis-ci.com/cloudalchemy/ansible-node-exporter/builds/137379386

**Comment by paulfantom** [comment]
This is our CI output of molecule tests. You can run those locally by following https://github.com/cloudalchemy/ansible-node-exporter#local-testing.

What we need is to extend those to cover feature included in this PR, same way as it is happening in cloudalchemy/ansible-prometheus#239

**Comment by koroglusaban** [comment]
[moleculetest-ansible-node-exporter.txt](https://github.com/cloudalchemy/ansible-node-exporter/files/3874103/moleculetest-ansible-node-exporter.txt)

**Comment by paulfantom** [comment]
Sorry, but I don't know what I have to do with this file.

I was saying about adding molecule tests *in code* to this PR. Things like this:
- https://github.com/cloudalchemy/ansible-prometheus/blob/dc277d8db3c8d0682f4b2a9fc98df0cba7993938/molecule/alternative/prepare.yml
- https://github.com/cloudalchemy/ansible-prometheus/blob/dc277d8db3c8d0682f4b2a9fc98df0cba7993938/molecule/alternative/playbook.yml

----
Additionally. I just noticed that this PR is adding new variable to `defaults/main.yml`. This needs to be reflected in `README.md` file in `Role Variables` section.

**Comment by paulfantom** [comment]
Closing as this was already done in #126

**Pull request closed without merging**